packages/web/src/pages/upload-page/forms/EditTrackForm.tsx

```diff
@@ -80,7 +80,9 @@ export const EditTrackForm = (props: EditTrackFormProps) => {
           commercialUse: null,
           derivativeWorks: null
         },
-        stems: []
+        stems: [],
+        isrc: '',
```

Comment:
this was needed for the text-input not to complain, not sure if this will lead to issues with the actual upload?

Reply:
yeah let's ship this and see if it causes issues